### DISCUSSION TITLE
How to add WindiCSS plugins like "animations"

### DISCUSSION THREAD

**Discussion by nvhoang55** [unocss/unocss#996] [Q&A] [answered]
Like the title, does anyone know how to apply WindiCSS plugin like "animations" with UnoCSS?

**Comment by userquin** [comment, marked as answer]
On preset uno you have the animate stuff: https://uno.antfu.me/?s=animate

**Reply by nvhoang55** [reply]
Thank you so much, I can use animate via UnoCSS now
